spec/models/acl_spec.rb

```diff
@@ -0,0 +1,44 @@
+require 'rails_helper'
```

Comment:
@dtelaroli after remove the scaffold generated tests on dummy app, I will focus on write specs for the actual codebase and a few small changes on the code.